src/components/activities/content/editors/SyllableEditor.tsx

```diff
@@ -0,0 +1,67 @@
+import ContentEditable from 'react-contenteditable';
+import React from 'react';
+
+import type { AnthemData } from 'src/activity-types/anthem.types';
+import BacklineIcon from 'src/svg/anthem/backline.svg';
+import TrashIcon from 'src/svg/anthem/trash.svg';
+import type { Activity } from 'types/activity.type';
+
+type SyllableEditorProps = {
+  update: (activity: Partial<Activity>) => void;
+  index: number;
+  backline?: boolean;
+  editable?: boolean;
+  data: AnthemData;
+};
+
+export const SyllableEditor: React.FC<SyllableEditorProps> = ({
+  update,
+  index,
+  backline = false,
+  editable = false,
+  data,
+}: React.PropsWithChildren<SyllableEditorProps>) => {
```

Comment:
**Suggestion**: Remove `React.PropsWithChildren`, you are not using children in this component.